pandas/tests/series/indexing/test_datetime.py

```diff
@@ -427,25 +427,27 @@ def test_indexing():
     # getting
 
     # GH 3070, make sure semantics work on Series/Frame
-    expected = ts["2001"]
-    expected.name = "A"
+    result = ts["2001"]
+    tm.assert_series_equal(result, ts.iloc[:12])
```

Comment:
This `expected` wasn't actually being used right now, and it's the result of the "getting" operation (with a timeseries partial indexing), so asserting the result instead.